packages/examples/cvat/exchange-oracle/src/crons/process_recording_oracle_webhooks.py

```diff
@@ -83,29 +81,22 @@ def handle_recording_oracle_event(webhook: Webhook, *, db_session: Session, logg
                 for project in projects_chunk:
                     if project.status != ProjectStatuses.validation:
                         logger.error(
-                            "Unexpected event {} received for a project in the {} status, "
-                            "ignoring (escrow_address={}, project={})".format(
-                                webhook.event_type,
-                                project.status,
-                                webhook.escrow_address,
-                                project.cvat_id,
-                            )
+                            f"Unexpected event {webhook.event_type} received for a project in the {project.status} status, "
+                            f"ignoring (escrow_address={webhook.escrow_address}, project={project.cvat_id})"
                         )
                         return
 
                     new_status = ProjectStatuses.recorded
                     logger.info(
-                        "Changing project status to {} (escrow_address={}, project={})".format(
-                            new_status, webhook.escrow_address, project.cvat_id
-                        )
+                        f"Changing project status to {new_status} (escrow_address={webhook.escrow_address}, project={project.cvat_id})"
                     )
                     cvat_db_service.update_project_status(db_session, project.id, new_status)
 
         case RecordingOracleEventTypes.task_rejected:
             event = RecordingOracleEvent_TaskRejected.parse_obj(webhook.event_data)
 
             rejected_jobs = cvat_db_service.get_jobs_by_cvat_id(db_session, event.rejected_job_ids)
-            rejected_project_cvat_ids = set(j.cvat_project_id for j in rejected_jobs)
+            rejected_project_cvat_ids = {j.cvat_project_id for j in rejected_jobs}
```

Comment:
I don't really like this specific change for set declarations. With `set` it's impossible to confuse the declaration with a dict, so the code intention is clear. With `{}` you need to actually parse the expression to understand what's inside the variable, and you don't have any information about the author intentions for the variable.

Reply:
Reverted the changes and disabled the rule for now.